test/unit/stop_test.lua

```diff
@@ -0,0 +1,34 @@
+local expirationd = require("expirationd")
```

Comment:
Could you please rename file with test to something like `task_test.lua` or `task_stop_test.lua`. Right now it is unclear what kind of stop we test here without looking to a file content. We plan to move TAP tests to luatest format and it would be good to give a clear names for files.

Reply:
Renamed to `task_stop_test.lua`